README.md

```diff
@@ -1,23 +1,43 @@
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-11-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-This repository hosts Quarkus extensions for different Azure Services. You can check the [documentation of these services](https://quarkiverse.github.io/quarkiverse-docs/quarkus-azure-services/dev/index.html).
+This repository hosts Quarkus extensions for different Azure Services. You can check
+the [documentation of these services](https://quarkiverse.github.io/quarkiverse-docs/quarkus-azure-services/dev/index.html).
 
 ## Azure Services
 
 The following extensions allows you to interact with some of the Azure Services:
 
-- [Quarkus Azure App Configuration Extension](https://quarkiverse.github.io/quarkiverse-docs/quarkus-azure-services/dev/quarkus-azure-app-configuration.html): [Azure App Configuration](https://azure.microsoft.com/products/app-configuration) is a fast, scalable parameter storage for app configuration.
-- [Quarkus Azure Blob Storage Extension](https://quarkiverse.github.io/quarkiverse-docs/quarkus-azure-services/dev/quarkus-azure-storage-blob.html): [Azure Blob Storage](https://azure.microsoft.com/products/storage/blobs/) is a massively scalable and secure object storage for cloud-native workloads, archives, data lakes, high-performance computing, and machine learning.
+- [Quarkus Azure App Configuration Extension](https://quarkiverse.github.io/quarkiverse-docs/quarkus-azure-services/dev/quarkus-azure-app-configuration.html): [Azure App Configuration](https://azure.microsoft.com/products/app-configuration)
+  is a fast, scalable parameter storage for app configuration.
+- [Quarkus Azure Blob Storage Extension](https://quarkiverse.github.io/quarkiverse-docs/quarkus-azure-services/dev/quarkus-azure-storage-blob.html): [Azure Blob Storage](https://azure.microsoft.com/products/storage/blobs/)
+  is a massively scalable and secure object storage for cloud-native workloads, archives, data lakes, high-performance
+  computing, and machine learning.
 
 ## Example applications
 
 Example applications can be found inside the [integration-tests](integration-tests) folder:
 
-- [Azure App Configuration sample](integration-tests/azure-app-configuration): REST endpoint using the Quarkus extension to get the configuration stored in Azure App Configuration.
-- [Azure Blob Storage samples](integration-tests/azure-storage-blob): REST endpoint using the Quarkus extension to upload and download files to/from Azure Blob Storage.
+- [Azure App Configuration sample](integration-tests/azure-app-configuration): REST endpoint using the Quarkus extension
+  to get the configuration stored in Azure App Configuration.
+- [Azure Blob Storage sample](integration-tests/azure-storage-blob): REST endpoint using the Quarkus extension to
+  upload and download files to/from Azure Blob Storage.
+
+## Release
```

Comment:
I think it's better to describe the release date and release version here.

Reply:
How about using pattern like [this](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/CHANGELOG.md#release-history).
<img width="548" alt="image" src="https://github.com/quarkiverse/quarkus-azure-services/assets/4465723/d6e1b3f2-1aee-4e67-bd1b-81a41d850f24">

Reply:
This section is not designed for describing releases, instead, it's for telling the repo maintainers how to release a new version. I updated the title to clarify the intention, see b17ab98.